language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionContextTypeResolver.java

```diff
@@ -34,6 +34,7 @@
 /**
  * This visitor is used to resolve the type of given code action context.
  * 
+ * Todo: Use the ContextTypeResolver instead.
```

Comment:
@malinthar Did we create an issue for this?

Reply:
I will create an issue.

Reply:
Thanks!